testdata/testinput2

```diff
@@ -831,6 +831,16 @@
 
 /x++/IB
 
+# For comparison with the following test, which disables auto-possessification
```

Comment:
would be nice for these to also have one with the "legacy" option

Reply:
and of course by "these", I meant all three similar options.

Reply:
Done.